cumulusci/cumulusci.yml

```diff
@@ -51,18 +51,6 @@ tasks:
             managed: True
             namespace_token: '%%%NAMESPACE%%%'
             filename_token: '___NAMESPACE___'
-    dx_push:
-        description: Uses Salesforce DX to push code to a scratch workspace org
-        class_path: cumulusci.tasks.salesforcedx.BaseSalesforceDXTask
-        options:
-            command: force:src:push
-            options: '-a'
-    filter_subscribers:
```

Comment:
filter_subscribers and get_subscribers both raise NotImplemented errors from the task class. I'm not sure why we mapped those two through in the cumulusci.yml.  I'm going to remove both from this branch.  I'll put the generate_apex_docs task back though.  This was caused by a merge conflict.